pkg/utils/netlink_provider.go

```diff
@@ -48,6 +50,19 @@ func GetNetlinkProvider() NetlinkProvider {
 	return netlinkProvider
 }
 
+// HasRdmaParam returns true if PCI device has "enable_rdma" param
+// equivalent to "devlink dev param show pci/0000:d8:01.1 name enable_rdma"
+func (defaultNetlinkProvider) HasRdmaParam(pciAddr string) (bool, error) {
+	param, err := nl.DevlinkGetDeviceParamByName("pci", pciAddr, "enable_rdma")
+	if err != nil {
+		return false, fmt.Errorf("error getting enable_rdma attribute for pci device %s %v", pciAddr, err)
+	}
+	if len(param.Values) == 0 || param.Values[0].Data == nil {
+		return false, nil
+	}
+	return true, nil
```

Comment:
is there a case where enable_rdma parameter value has data which interprets to "false" ?

Reply:
Returning the value of the param now.